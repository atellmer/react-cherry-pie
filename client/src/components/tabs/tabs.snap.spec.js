import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

import TmTabs from './index';


const props = {
  interlocutor: {},
  changeInterlocutor: () => {},
  dialogs: []
};

test('Jest: TmTabs (Snapshot)', () => {
  const shallowRenderer = ReactTestUtils.createRenderer();

  const result = shallowRenderer.render(<TmTabs {...props}/>);
  expect(result).toMatchSnapshot();
});
