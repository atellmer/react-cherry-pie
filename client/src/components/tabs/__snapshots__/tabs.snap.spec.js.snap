exports[`test Jest: TmTabs (Snapshot) 1`] = `
<div
  className={undefined}>
  <Tabs
    initialSelectedIndex={0}
    onChange={[Function]}
    value={0}>
    <Tab
      label="Все"
      value={0} />
    <Tab
      label="Новые"
      value={1} />
  </Tabs>
  <SwipeableViews
    animateHeight={false}
    animateTransitions={true}
    axis="x"
    disabled={false}
    index={0}
    onChangeIndex={[Function]}
    resistance={false}
    springConfig={
      Object {
        "damping": 30,
        "stiffness": 300,
      }
    }
    threshold={5}>
    <Scrollbars
      autoHeight={false}
      autoHeightMax={200}
      autoHeightMin={0}
      autoHide={true}
      autoHideDuration={200}
      autoHideTimeout={1000}
      className={undefined}
      hideTracksWhenNotNeeded={false}
      renderThumbHorizontal={[Function]}
      renderThumbVertical={[Function]}
      renderTrackHorizontal={[Function]}
      renderTrackVertical={[Function]}
      renderView={[Function]}
      tagName="div"
      thumbMinSize={30}
      universal={false}>
      
    </Scrollbars>
    <Scrollbars
      autoHeight={false}
      autoHeightMax={200}
      autoHeightMin={0}
      autoHide={true}
      autoHideDuration={200}
      autoHideTimeout={1000}
      className={undefined}
      hideTracksWhenNotNeeded={false}
      renderThumbHorizontal={[Function]}
      renderThumbVertical={[Function]}
      renderTrackHorizontal={[Function]}
      renderTrackVertical={[Function]}
      renderView={[Function]}
      tagName="div"
      thumbMinSize={30}
      universal={false}>
      tab #2
    </Scrollbars>
  </SwipeableViews>
</div>
`;
